HISTORY.md

```diff
@@ -7,6 +7,7 @@
 
 ## Breaking behavior
 * Removes TBB as an optional dependency [#2181](https://github.com/TileDB-Inc/TileDB/pull/2181)
+* When an array has been opened using timestamps, if it is to be reopened without timestamps, the timestamps and config object must be reset first. [#2142](https://github.com/TileDB-Inc/TileDB/pull/2142)
```

Comment:
I appreciate this being documented, but I think it is too much of a corner case to document here.